jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/actions/DeleteQueueAction.kt

```diff
@@ -15,9 +15,10 @@ import software.aws.toolkits.resources.message
 
 class DeleteQueueAction : DeleteResourceAction<SqsQueueNode>(message("sqs.delete.queue.action"), TaggingResourceType.SQS_QUEUE) {
     override fun performDelete(selected: SqsQueueNode) {
-        val client = selected.nodeProject.awsClient<SqsClient>()
-        SqsWindow.getInstance(selected.nodeProject).closeQueue(selected.queueUrl)
+        val project = selected.nodeProject
+        val client = project.awsClient<SqsClient>()
+        SqsWindow.getInstance(project)?.closeQueue(selected.queueUrl)
```

Comment:
Why did getIsntance start returning null?

Reply:
Changed to use the extension function